catalog/models.py

```diff
@@ -1,3 +1,79 @@
 from django.db import models
+from django.urls import reverse
+import uuid # Required for unique book instances
 
 # Create your models here.
+class Genre(models.Model):
+    """Model representing a book genre."""
+    name = models.CharField(max_length=20, help_text='Enter a book genre (e.g.Science Fiction)')
+
+    def __str__(self):
+        """String for representing the Model object."""
+        return self.name
+
+class Book(models.Model):
+    """Model representing a book (but not a specific copy of a book)."""
+    title = models.CharField(max_length=200)
+    author = models.ForeignKey('Author', on_delete=models.SET_NULL, null=True)
+    
+    summary = models.TextField(max_length=1000, help_text='Enter a brief description of the book')
+    isbn = models.CharField('ISBN', max_length=13, unique=True,help_text='13 Character <ahref="https://www.isbn-international.org/content/what-isbn">ISBN number</a>')
+    
+    genre = models.ManyToManyField(Genre, help_text='Select a genre for this book')
```

Comment:
Khai báo dòng này thì khi migrate sẽ được gì e?

Reply:
Ta sẽ tạo được một bảng gồm các cột summary, isbn cho book modelvà đồng thời tạo mối quan hệ many to many với  Genre model ạ